src/Components/MeetingDetails.tsx

```diff
@@ -0,0 +1,355 @@
+import React from "react";
+import { BiVideo } from "react-icons/bi";
+import { BiCircle } from "react-icons/bi";
+
+type MeetingDetailsProps = {
+  meetingId: number | null;
+};
+
+const meetings = [
+  {
+    id: 1,
+    heading: "Virtual Meeting Notes",
+    title: "Meeting Title: Weekly Team Check-In",
+    date: "Date: April 15, 2024",
+    time: "Time: 10:00 AM - 11:00 AM (PST)",
+    Attendees: ["[Name 1]", "[Name 2]", "[Name 3]", "[Name 4]"],
+    agenda: [
+      "Project Status Updates",
+      "Discussion on Upcoming Deadlines",
+      "Action Items and Follow-ups",
+    ],
+    notes: [
+      {
+        section: "Project Status Updates",
+        items: [
+          "[Name 1] provided an update on [Project A]. [Key points and decisions discussed].",
+          "[Name 2] discussed the progress of [Project B]. [Key points and decisions discussed].",
+        ],
+      },
+      {
```

Comment:
I suggest you add this data in separate js file and import it. To keep your code clean. This will be deprecated once we have the backend up